app/services/ClickHandler.coffee

```diff
@@ -0,0 +1,69 @@
+$ = require 'jquery'
+
+class ClickHandler
+
+  constructor: (@board, two, @clicked=[]) ->
```

Comment:
I've always thought this was how you defined optional parameters in coffeescript?